articles/libraries/machine-learning/basic-classification.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Quantum machine learning library
+author: geduardo
+ms.author: v-edsanc@microsoft.com
+ms.date: 02/16/2020
+ms.topic: article
+uid: microsoft.quantum.libraries.machine-learning.basics
+---
+
+# Basic classification: classify wines with the QDK
+
+In this Quickstart, you will learn how to execute a quantum sequential classifier written in Q# using the Quantum Machine Learning library of the QDK. It's okay if you don't understand everything, this is a quick overview of a complete QDK classifier. You will get more details explained as you go through the rest of the tutorials.
+
+In this guide we will use the [wine dataset](https://archive.ics.uci.edu/ml/datasets/wine) from the [UCI Machine Learning Repository](https://archive.ics.uci.edu/ml/datasets/wine), using a classifier structure defined in Q#. This dataset is included in the collection of standard datasets in the Quantum Machine Learning library. This means that we can load it directly from Q#.
+
+## Execute the classifier from the host program
+
+First, let's take a look at the host programs to see how the process looks like. It consists of three parts:
+
+- Initialization of the target machine, in this case a quantum simulator.
+- Execution of the training to obtain the parameters and the bias that determine the model.
+- Validation of the model to test the accuracy.
+
+    ### [Python with Visual Studio Code or the Command Line](#tab/tabid-python)
+
+    To run your the Q# classifier from Python, save the following code as `host.py`. Remember that you also need the Q# file `Training.qs` that is explained later in this tutorial.
+
+    code language="python" source="~/quantum/samples/machine-learning/wine/host.py" range="9-14"
```

Comment:
Do you need the `:::` notation here? The Python hosts are up with microsoft/Quantum#317, by the way.

Reply:
Yup, I removed them meanwhile becaues I was having compilation errors. Although I don't know if they were related at all, I think it was something with the backend of the docs.